packages/react-instantsearch/src/__tests__/common-widgets.test.tsx

```diff
@@ -191,6 +191,16 @@ const testSetups: TestSetupsMap<TestSuites> = {
       </InstantSearch>
     );
   },
+  createHitsWidgetOptionsTests({ instantSearchOptions, widgetParams }) {
```

Comment:
why is this separate from the main hits tests?

Reply:
That was my initial setup to avoid messing up with the Insights tests. Changed in [f8d3652](https://github.com/algolia/instantsearch/pull/5870/commits/f8d36520e03cfa67d9019e6c98ae3d0c361b20cc).